### PR TITLE
replace size_average param

### PR DESCRIPTION
**Pull request by breid1313**
The size_average parameter is being depreciated. The preferred way to handle loss summation/averaging is now:

size_average=True => reduction='mean'
size_average=False => reduction='false'

documentation is [here](https://pytorch.org/docs/stable/nn.html?highlight=nll%20loss#torch.nn.NLLLoss)

### CODE REVIEW

**Comment by breid1313** [comment]
Cleaning up my PRs. Closing due to lack of repo activity.
